splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -1815,6 +1815,14 @@
               "os"
             ]
           }
+        },
+        "supportedThemes": {
+          "description": "This feature allows you to choose theme based on the given option i.e. light and dark",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": ["light", "dark"]
+          }
```

Comment:
If it is not provided in the globalConfig file, `app.conf[ui].supported_themes` would not be filled?

Reply:
Yes, it would not be filled as it is an optional parameter.